integration-tests/profiler/profiler.spec.js

```diff
@@ -201,18 +201,19 @@ class FilesystemEventProcessor extends TimelineEventProcessor {
   }
 }
 
-async function gatherFilesystemTimelineEvents (cwd, scriptFilePath) {
-  return gatherTimelineEvents(cwd, scriptFilePath, 'fs', [], FilesystemEventProcessor)
+async function gatherFilesystemTimelineEvents (cwd, scriptFilePath, agentPort) {
+  return gatherTimelineEvents(cwd, scriptFilePath, agentPort, 'fs', [], FilesystemEventProcessor)
 }
 
-async function gatherTimelineEvents (cwd, scriptFilePath, eventType, args, Processor) {
+async function gatherTimelineEvents (cwd, scriptFilePath, agentPort, eventType, args, Processor) {
   const procStart = BigInt(Date.now() * 1000000)
   const proc = fork(path.join(cwd, scriptFilePath), args, {
     cwd,
     env: {
       DD_PROFILING_EXPORTERS: 'file',
       DD_PROFILING_ENABLED: 1,
-      DD_INTERNAL_PROFILING_TIMELINE_SAMPLING_ENABLED: 0 // capture all events
+      DD_INTERNAL_PROFILING_TIMELINE_SAMPLING_ENABLED: 0, // capture all events
```

Comment:
You could also disable remote config with `DD_REMOTE_CONFIGURATION_ENABLED: 0`

Reply:
I did, and that alone didn't help. Something else (maybe trace result posting) is still keeping the process alive for about 25-30 seconds.